app/controllers/forms_controller.rb

```diff
@@ -22,6 +23,12 @@ def show
   end
 
   def submit
+    if @current_user.present?
+      @current_user.update_attributes(
+        params.require(:user)
+        .permit(:first_name, :last_name, :email)
+      )
+    end
```

Comment:
Right, I just mean invalid in terms of the validations we place on those fields.